client/container.go

```diff
@@ -662,7 +642,7 @@ type ResContainerSetEACL struct {
 //
 // Return statuses:
```

Comment:
No longer relevant. Or maybe just should reference `apistatus`.

Reply:
Removed